test/e2e/fixtures/releases/test/fail-no-rollback.yaml

```diff
@@ -9,6 +9,7 @@ spec:
   timeout: 30
   test:
     enable: true
+    ignoreFailures: true
```

Comment:
looks like this file was actually unused before. can we rename this file to indicate that it is about ignoring test failures e.g. `fail-ignored.yaml`?

also can we enable rollbacks in this file to show that the `ignoreFailures` and not rollbacks being disabled is what is causing the rollback not to happen. i'm wondering if that may actually be what's causing the test to pass currently as I'm not seeing any code in `release.go` to handle avoiding rollbacks / uninstalls. speaking of, can we also add a test to ensure un-installs don't happen after an install when `ignoreFailures` is set?

Reply:
> can we also add a test to ensure un-installs don't happen after an install when ignoreFailures is set?

Have you got any suggestions how I can assert that an uninstall has not happened as the current test infrastructure only supports waiting for a condition to be become true and not lack of a condition occurring?

Reply:
> can we rename this file to indicate that it is about ignoring test failures 
done

> also can we enable rollbacks in this file to show that the ignoreFailures and not rollbacks being disabled is what is causing the rollback not to happen
done

>  i'm wondering if that may actually be what's causing the test to pass currently as I'm not seeing any code in release.go to handle avoiding rollbacks / uninstalls

code was lost in merge/commit. clearly the test did not provide enough coverage so added an extra assertion to check it is not rolled back

> can we also add a test to ensure un-installs don't happen after an install when ignoreFailures is set?

I have added an extra assertion to the existing test to validate it ignores the test failure